src/frontend/src/lib/services/load-user-profile.services.ts

```diff
@@ -0,0 +1,72 @@
+import type { UserProfile } from '$declarations/backend/backend.did';
+import { createUserProfile, getUserProfile } from '$lib/api/backend.api';
+import { i18n } from '$lib/stores/i18n.store';
+import { userProfileStore } from '$lib/stores/settings.store';
+import { toastsError } from '$lib/stores/toasts.store';
+import { UserProfileNotFoundError } from '$lib/types/errors';
+import type { Identity } from '@dfinity/agent';
+import { isNullish } from '@dfinity/utils';
+import { get } from 'svelte/store';
+
+const queryProfile = async ({
+	identity,
+	certified
+}: {
+	identity: Identity;
+	certified: boolean;
+}): Promise<UserProfile> => {
+	const response = await getUserProfile({ identity, certified });
+	if ('Ok' in response) {
+		return response.Ok;
+	}
+	const err = response.Err;
+	if ('NotFound' in err) {
+		throw new UserProfileNotFoundError();
+	}
+	throw new Error('Unknown error');
+};
+
+const queryUnsaveProfile = async ({
```

Comment:
typo? `unsave`

Reply:
No... I expect `unsave` to be a word and this is camelcase.

Reply:
Ah. Well, I've got no idea what `unsave` means in this context? I understand it as "query the list of profile that have been reverted from being saved (unsaved)"

Reply:
ah, sorry, hahaha, it should be Unsafe, the problem is the "f", I thought you were saying that it should be a lower case "u", not "U" 😂

Reply:
hahaha glad my english is not that bad 😅